src/client.js

```diff
@@ -78,6 +94,9 @@ class SkynetClient {
       setEntryData: browserClient.dbV2.setEntryData.bind(browserClient),
       deleteEntryData: browserClient.dbV2.deleteEntryData.bind(browserClient),
       getRawBytes: browserClient.dbV2.getRawBytes.bind(browserClient),
+
+      // Holds the cached revision numbers, protected by mutexes to prevent
+      // concurrent access.
```

Comment:
Hello m-cat,

If I'm precise, I should have made the comment at the PR of "skydb".
I had already done it, but it was integrated differently than in "skynet-js", and that's why I took it out again and left it like that.

But now I thought it would be better to add it again so that it looks more original to "skynet-js".
If you didn't want it, I would have taken it out again.

You may see that I added "deprecated" to "SkyDB v1" as in "skynet-js".